distributed/security.py

```diff
@@ -60,13 +60,15 @@ class Security:
         "tls_worker_cert",
     )
 
-    def __init__(self, **kwargs):
+    def __init__(self, require_encryption=None, **kwargs):
         extra = set(kwargs).difference(self.__slots__)
         if extra:
             raise TypeError("Unknown parameters: %r" % sorted(extra))
-        self._set_field(
-            kwargs, "require_encryption", "distributed.comm.require-encryption"
-        )
+        if require_encryption is None:
+            require_encryption = dask.config.get("distributed.comm.require-encryption")
+        if require_encryption is None:
+            require_encryption = not not kwargs
```

Comment:
Can you explain what this line does?

Reply:
I suspect this is a mistake ? `not not` ?

Reply:
`not not` turns anything into True/False if it is truthy/falsey.

```python

In [1]: not not {"x": 1}                                                                                                                                                                                          
Out[1]: True

In [2]: not not {}                                                                                                                                                                                                
Out[2]: False
```

Reply:
Probably calling `bool` is better

Reply:
Oh!  Cool!  And we do that in other places throughout the dask ecosystem.  

So @samaust this line is saying any time the user is setting security settings (in kwargs) dask will now enable `require_encryption` without the user explicitly setting it.  Essentially the title of the PR.